honeybee_doe2/util.py

```diff
@@ -4,7 +4,8 @@
 
 import re
 import os
-
+from collections import defaultdict
+from pathlib import Path
```

Comment:
You don't need to make any changes here but FYI, I do occasionally use IronPython 2.7 inside Rhino/Grasshopper to debug people's models and `pathlib` will give you an ImportError over there. So I hope you don't mind that I may tweak your code after I merge it just to make it run-able in Python 2.7 through 3.12. I know that 2.7-3.12 is an unreasonable range of Python versions over which to expect code contributions to function. So definitely let me take care of it. I just wanted to give you a head's up about it that doesn't mean I think the code is bad.